tmva/CMakeLists.txt

```diff
@@ -17,7 +17,9 @@ if(r OR tmva-rmva)
    add_subdirectory(rmva)
 endif()
 
+add_subdirectory(sofie)
```

Comment:
Does this mean that turning off sofie still builds sofie?

Reply:
This is correct, I think will be better to rename the flag tmva-onnx

Reply:
I will do this in a separate PR, for the time being is better merging this to fix the master